docs/rules/no-inner-declarations.md

```diff
@@ -137,6 +137,10 @@ if (test) {
 function doAnotherThing() {
     var baz = 81;
 }
+
+if (foo) var a;
+
+if (foo) function f(){}
```

Comment:
These should be in "incorrect examples".

Maybe `if (foo) function f(){}` in incorrect for "functions", `if (foo) var a;` in incorrect for "both".

Reply:
oh shoot, I added them in wrong section. I will change it